### ISSUE TITLE
29461161: Failures for iOS UI Tests in CI due to "App Accessibility Isn't Loaded"

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16445]
#### Description

Summary:
When running iOS XCUI Tests in CI (Bitrise for the attached file, but it's not limited to Bitrise), tests hang for one minute and then fail due to "app accessibility isn't loaded".

-
Product Version: XC 8.2 beta
Created: 2016-12-01T18:28:54.577910
Originated: 2016-12-01T00:00:00
Open Radar Link: http://www.openradar.me/29461161
